sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -891,4 +894,19 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     const ruleResponse: RuleResponse = Object.assign(rule || {}, { _response: response });
     return ruleResponse;
   }
+
+  private buildResponse(fn: Function, response: HttpOperationResponse): any {
+    try {
+      return fn(response);
+    } catch (err) {
+      log.error(`Failure parsing response from service - ${err}`);
```

Comment:
I recall there being a difference in the way errors get rendered when using
`log.error(`Failure parsing response from service - ${err}`)` vs `log.error(`Failure parsing response from service`, err)`.
Or maybe it was whether JSON.stringify() was used on the error vs passing the error object as a separate argument.

Please try out both and pick one that logs the entire error correctly

Reply:
Updated